composer.json

```diff
@@ -12,24 +12,25 @@
         }
     ],
     "require": {
-        "symfony/framework-bundle": "~2.1",
-        "symfony/process": "~2.1",
-        "symfony/finder": "~2.1",
+        "php": "^5.5|~7.0",
+        "symfony/framework-bundle": "~2.8|~3.0",
```

Comment:
I do not agree during that 2.3 is maintained by the core team, I think that we have to maintain it too.
It is not so complicated to support both versions...